float64[] waypoint_times
float64[] waypoint_x
float64[] waypoint_y
float64 velocity